### PR TITLE
Add PHP support for showcase testing

### PR DESCRIPTION
**Pull request by michaelbausor**

### CODE REVIEW

**Comment by michaelbausor** [comment]
@andreamlin @landrito  I am having trouble setting the SHOWCASE_VERSION env variable and have it be picked up correctly. The task to run the showcase server works in the latest commit where I have hardcoded the value, but all previous commits it couldn't see the environment variable and I'm not sure  why. Any ideas?

**Comment by michaelbausor** [comment]
@landrito @andreamlin This is ready for review, PTAL

**Comment by andreamlin** [comment]
One question: the "Run Showcase Server" job in the CI run https://circleci.com/gh/googleapis/gapic-generator/4585 was canceled because 
```
2018/09/21 20:13:18 Received Request for Method: /google.showcase.v1alpha1.Echo/Echo
2018/09/21 20:13:18     Request:  error:<code:3 message:"Unary error message" > 
2018/09/21 20:13:18     Returning Error: rpc error: code = InvalidArgument desc = Unary error message
2018/09/21 20:13:18 
2018/09/21 20:13:18 Got signal: interrupt
Daemon was interruped by system signal
Step was canceled
```

Is this supposed to happen?

**Comment by michaelbausor** [comment]
Yes I believe so. The error is from the error test, which is expected, while the interrupt is just circleci shutting down the job once everything else is done. At least, this is my understanding of the situation, @landrito can confirm if this is correct.

**Comment by landrito** [comment]
Correct it is just getting the interrupt from circleci closing the job.

File: .circleci/config.yml [hunk 1]
```diff
@@ -234,6 +259,31 @@ anchor_run_php: &anchor_run_php
...
+        command: |
+          echo 'export TEST_API="showcase-v1alpha1"' >> $BASH_ENV
+          echo 'export KEY_REQUIRED=false' >> $BASH_ENV
+    - run:
+        name: Test Showcase Generated Unit Tests.
+        <<: *anchor_test_php
+    - run:
+        name: Run Showcase Server.
+        command: |
+          ls
```
Comment (landrito) [inline .circleci/config.yml]: nit: remove debugging prints.

File: showcase/php/composer.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  ],
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "require": {
+    "google/showcase": "*"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^4.8|^5.0"
+  }
+}
```
Comment (landrito) [inline showcase/php/composer.json]: nit: endline

File: showcase/php/phpunit.xml.dist [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    convertNoticesToExceptions="true"
+    convertWarningsToExceptions="true"
+    strict="true"
+    >
+  <testsuites>
+    <testsuite>
+      <directory>tests</directory>
+    </testsuite>
+  </testsuites>
+</phpunit>
```
Comment (landrito) [inline showcase/php/phpunit.xml.dist]: nit: endline

File: .circleci/config.yml (hunk 1 above)
Comment (michaelbausor) [reply .circleci/config.yml]: Fixed

File: showcase/php/composer.json (hunk 2 above)
Comment (michaelbausor) [reply showcase/php/composer.json]: Fixed

File: showcase/php/phpunit.xml.dist (hunk 3 above)
Comment (michaelbausor) [reply showcase/php/phpunit.xml.dist]: Fixed

**Pull request merged**